test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -51,7 +51,13 @@ private static int Run(Options opts)
             try
             {
                 s_repository = opts.Repository;
-                s_repositoryName = $"{(opts.DryRun ? "dryrun." : "")}{Path.GetFileName(opts.Repository)}";
+
+                s_repositoryName = opts.DryRun
```

Comment:
rename to `s_testName`, the original name could be confusing in the future.

Reply:
Done.